util/topsql/reporter/reporter_test.go

```diff
@@ -245,6 +245,10 @@ func TestCollectAndTopN(t *testing.T) {
 	require.Equal(t, 5, getTotalCPUTime(results[1]))
 	require.Equal(t, []byte("sqlDigest3"), results[2].SqlDigest)
 	require.Equal(t, 3, getTotalCPUTime(results[2]))
+	// sleep for wait all SQL meta received finish.
```

Comment:
```suggestion
	// sleep to wait for all SQL meta received.
```

Reply:
BTW, I suggest using some internal conditions to determine the timing of test, not waiting for a fixed period of time, which could introduce uncertainty into the test.